### ISSUE TITLE
Raspiblitz fail at startup

### ISSUE THREAD

**Issue by Minituff** [mempool/mempool#354]
### Description

I am getting a `**Waiting for blocks...**` error on mempool running on Raspiblitz 1.6.3

If this issue is fixed in a newer version of mempool, how would I update without waiting for Raspiblitz to update themselves, which could take a few weeks/months?

#### Version
Mempool v2.0.1
This is the version included with Raspiblitz

### Steps to reproduce

To get the debug output, I went into the backend folder and ran `nmp run start`.

### Debug Output

```bash
admin@raspberrypi:/home/mempool/mempool/backend $ npm run start

> mempool-backend@2.0.1 start /home/mempool/mempool/backend
> node --max-old-space-size=4096 dist/index.js


<--- Last few GCs --->

[18485:0x3991890]      235 ms: Mark-sweep 1.0 (3.5) -> 1.0 (3.5) MB, 5.9 / 0.0 ms  (average mu = 0.554, current mu = 0.031) allocation failure GC in old space requested
[18485:0x3991890]      240 ms: Mark-sweep (reduce) 1.0 (3.5) -> 1.0 (3.5) MB, 3.6 / 0.0 ms  (average mu = 0.466, current mu = 0.181) last resort GC in old space requested
[18485:0x3991890]      244 ms: Mark-sweep (reduce) 1.0 (2.5) -> 1.0 (3.5) MB, 4.6 / 0.0 ms  (average mu = 0.315, current mu = 0.008) last resort GC in old space requested


<--- JS stacktrace --->

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
Aborted
npm ERR! code ELIFECYCLE
npm ERR! errno 134
npm ERR! mempool-backend@2.0.1 start: `node --max-old-space-size=4096 dist/index.js`
npm ERR! Exit status 134
npm ERR!
npm ERR! Failed at the mempool-backend@2.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/admin/.npm/_logs/2021-02-23T00_13_52_420Z-debug.log
admin@raspberrypi:/home/mempool/mempool/backend $ cat  /home/admin/.npm/_logs/2021-02-23T00_13_52_420Z-debug.log
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/lib/nodejs/node-v14.15.4-linux-armv7l/bin/node',
1 verbose cli   '/usr/bin/npm',
1 verbose cli   'run',
1 verbose cli   'start'
1 verbose cli ]
2 info using npm@6.14.10
3 info using node@v14.15.4
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle mempool-backend@2.0.1~prestart: mempool-backend@2.0.1
6 info lifecycle mempool-backend@2.0.1~start: mempool-backend@2.0.1
7 verbose lifecycle mempool-backend@2.0.1~start: unsafe-perm in lifecycle true
8 verbose lifecycle mempool-backend@2.0.1~start: PATH: /usr/local/lib/nodejs/node-v14.15.4-linux-armv7l/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/mempool/mempool/backend/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games:/sbin:/usr/local/lib/nodejs/node--/bin/
9 verbose lifecycle mempool-backend@2.0.1~start: CWD: /home/mempool/mempool/backend
10 silly lifecycle mempool-backend@2.0.1~start: Args: [ '-c', 'node --max-old-space-size=4096 dist/index.js' ]
11 silly lifecycle mempool-backend@2.0.1~start: Returned: code: 134  signal: null
12 info lifecycle mempool-backend@2.0.1~start: Failed to exec start script
13 verbose stack Error: mempool-backend@2.0.1 start: `node --max-old-space-size=4096 dist/index.js`
13 verbose stack Exit status 134
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/nodejs/node-v14.15.4-linux-armv7l/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/nodejs/node-v14.15.4-linux-armv7l/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid mempool-backend@2.0.1
15 verbose cwd /home/mempool/mempool/backend
16 verbose Linux 4.19.118-v7l+
17 verbose argv "/usr/local/lib/nodejs/node-v14.15.4-linux-armv7l/bin/node" "/usr/bin/npm" "run" "start"
18 verbose node v14.15.4
19 verbose npm  v6.14.10
20 error code ELIFECYCLE
21 error errno 134
22 error mempool-backend@2.0.1 start: `node --max-old-space-size=4096 dist/index.js`
22 error Exit status 134
23 error Failed at the mempool-backend@2.0.1 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 134, true ]
admin@raspberrypi:/home/mempool/mempool/backend $`
```

#### Device or machine

Raspberry Pi 4 (8gb)

#### Additional info

I tried manually changing the memory to 2048 as seen in this https://github.com/mempool/mempool/issues/339 issue but I am getting the same error:

**Comment by wiz** [comment]
This issue has already been fixed, and is pending a release by the RaspiBlitz team:
https://github.com/rootzoll/raspiblitz/pull/2007
https://github.com/rootzoll/raspiblitz/pull/2014

**Comment by wiz** [comment]
v1.7RC2 is available to test here with mempool v2.1.2: https://github.com/rootzoll/raspiblitz/blob/dev/README.md#downloading-the-software
